src/main/resources/templates/events/index.html

```diff
@@ -40,9 +40,9 @@
             <a href="#" th:href="@{/events/create}">Create an event</a>
         </p>
     </div>
+    <div class="alert alert-info" role="alert" th:unless="${#strings.isEmpty(message)}"><p th:text="${message}"></p>
```

Comment:
Hmm this isn't really part of the new feature, but the position of the alert was bugging me.